# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1361422973 18000
#      Thu Feb 21 00:02:53 2013 -0500
# Node ID 3a90c3f4046d11c268c7161ce45cf4297a4de87e
# Parent  9439f3b5c5fafe5580e70c48b578c139411f3b14
Fix typo in changeset dc39c1d84c5b

* configure.ac (CURLOPT_DIRLISTONLY): Delete extraneous closing bracket.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -867,17 +867,17 @@ if test -z "$warn_curl"; then
       ]], [[
       curl_easy_setopt ((CURL*)NULL, CURLOPT_DIRLISTONLY, 0);
       ]])],
       [octave_cv_header_define_curlopt_dirlistonly=no],
       [octave_cv_header_define_curlopt_dirlistonly=yes])
     ])
   if test $octave_cv_header_define_curlopt_dirlistonly = yes; then
     AC_DEFINE(CURLOPT_DIRLISTONLY, CURLOPT_FTPLISTONLY,
-      [Define to the legacy option name if using an older version of cURL.])]
+      [Define to the legacy option name if using an older version of cURL.])
   fi
 fi
 LIBS="$save_LIBS"
 CPPFLAGS="$save_CPPFLAGS"
 
 ### Check for either of Graphics/ImageMagick++ libraries
 
 AC_ARG_WITH([magick],
